Support building ponyc natively on arm64 Windows machines (#4689)
ponylang/ponyc: .release-notes/4689.md

## Support building ponyc natively on arm64 Windows

> PonyC can now be built on arm64 Windows.

Author: ```suggestion
Pony is now available natively for arm64 Windows.
```

> Pony is now available natively for arm64 Windows.

Author: ```suggestion
We've added support for building Pony for arm64 Windows.
```
